os/arch/arm/src/amebad/amebad_spi.c

```diff
@@ -0,0 +1,1237 @@
+ *
+ * Input Parameters:
+ *   priv   - private SPI device structure
+ *   offset - offset to the register of interest
+ *   value  - the 32-bit value to be written
+ *
+ * Returned Value:
+ *   The contents of the 32-bit register
+ *
+ ************************************************************************************/
+
+static inline void amebad_spi_putreg32(FAR struct amebad_spidev_s *priv,
+					uint8_t offset, uint32_t value)
+{
+	return;
+}
+
+/************************************************************************************
+ * Name: amebad_spi_readword
+ *
+ * Description:
+ *   Read one word from SPI
+ *
+ * Input Parameters:
+ *   priv - Device-specific state data
+ *
+ * Returned Value:
+ *   word as read
+ *
+ ************************************************************************************/
+
+static inline uint16_t amebad_spi_readword(FAR struct amebad_spidev_s *priv)
+{
+	return 0;
+}
+
+/************************************************************************************
+ * Name: amebad_spi_writeword
+ *
+ * Description:
+ *   Write one word to SPI
+ *
+ * Input Parameters:
+ *   priv - Device-specific state data
+ *   word - word to send
+ *
+ * Returned Value:
+ *   None
+ *
+ ************************************************************************************/
+
+static inline void amebad_spi_writeword(FAR struct amebad_spidev_s *priv, uint16_t word)
+{
+	return;
+}
+
+/************************************************************************************
+ * Name: amebad_spi_readbyte
+ *
+ * Description:
+ *   Read one byte from SPI
+ *
+ * Input Parameters:
+ *   priv - Device-specific state data
+ *
+ * Returned Value:
+ *   Byte as read
+ *
+ ************************************************************************************/
+
+static inline uint8_t amebad_spi_readbyte(FAR struct amebad_spidev_s *priv)
+{
+	return 0;
+}
+
+/************************************************************************************
+ * Name: amebad_spi_writebyte
+ *
+ * Description:
+ *   Write one 8-bit frame to the SPI FIFO
+ *
+ * Input Parameters:
+ *   priv - Device-specific state data
+ *   byte - Byte to send
+ *
+ * Returned Value:
+ *   None
+ *
+ ************************************************************************************/
+
+static inline void amebad_spi_writebyte(FAR struct amebad_spidev_s *priv, uint8_t byte)
+{
+	return;
+}
+
+/************************************************************************************
+ * Name: amebad_spi_9to16bitmode
+ *
+ * Description:
+ *   Check if the SPI is operating in more then 8 bit mode
+ *
+ * Input Parameters:
+ *   priv     - Device-specific state data
+ *
+ * Returned Value:
+ *   true: >8 bit mode-bit mode, false: <= 8-bit mode
+ *
+ ************************************************************************************/
+
+static inline bool amebad_spi_9to16bitmode(FAR struct amebad_spidev_s *priv)
+{
+	bool ret;
+
+	if (priv->nbits < 9) ret = false;
```

Comment:
How about like below?
```
if ( ) return false;
else return true;
```
Then we don't need local variable `ret`.

Reply:
Done as per suggestion.